product_docs/docs/pem/8/pem_inst_guide_linux/04_installing_postgres_enterprise_manager/06_configuring_the_pem_server_to_use_kerberos_authentication.mdx

```diff
@@ -99,7 +118,26 @@ chown enterprisedb <DATA_DIRECTORY_OF_POSTGRES>/<NAME_OF_PEM_DB_FILE>.keytab
 
     -   `POSTGRES_SERVICE_NAME` is the service name of Postgres (PostgreSQL/EDB Postgres Advanced Server) database. For eg: postgresql-13 for PostgreSQL 13 database on CentOS or RHEL platforms.
 
-## 5. Configure the PEM Server
+## 5. Obtain and view the initial ticket
+
+-   The `kinit` utility obtains and caches Kerberos tickets. This utility is typically used to obtain the ticket-granting ticket, using a password entered by the user to decrypt the credential from the key distribution center (KDC). The ticket-granting ticket is then stored in the user's credential cache.
+
+-   You can view the details of the ticket using `klist` utility.
+
+!!! Note
+    The `Kerberos Client` must be installed on the PEM Server and the Client machine to use `kinit` and `klist`.
+
+```text
+$ kinit <USERNAME@REALM>
+$ klist 
+```
+
+It will display the principal along with the kerberos ticket.
+
+!!! Note
+    The `USERNAME@REALM` specified here must be the database user having the `pem_admin` role and `CONNECT` privilege on `pem` database.
```

Comment:
```suggestion
    The `USERNAME@REALM` specified here must be a database user having the `pem_admin` role and `CONNECT` privilege on `pem` database.
```

Reply:
If there is only one user with the required role and privileges use "the" but if there could be more than one use "a"